aea/protocols/state_update/message.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""The state update message module."""
+
+from typing import Dict, cast
+
+from aea.protocols.base import Message
+
+TransactionId = str
+Address = str
+
+CurrencyEndowment = Dict[str, float]  # a map from identifier to quantity
+GoodEndowment = Dict[str, int]   # a map from identifier to quantity
+UtilityParams = Dict[str, float]   # a map from identifier to quantity
+ExchangeParams = Dict[str, float]   # a map from identifier to quantity
+
+
+class StateUpdateMessage(Message):
```

Comment:
Currently, I am thinking of using the same protocols internally for comms with the DM. Not sure if that makes sense though. Of course no serialization is needed